src/IK/include/BiomechanicalAnalysis/IK/InverseKinematics.h

```diff
@@ -135,6 +135,8 @@ class HumanIK
         int nodeNumber;
         manif::SO3d IMU_R_link; // Rotation matrix from the IMU to related link, set through config
                                 // file
+        manif::SO3d IMU_R_link_init; // Rotation matrix from the IMU to related link, set through config
+                                     // file
```

Comment:
The two descriptions are the same?

Reply:
I just copied from the line above and forgot to change it, fixed it with https://github.com/ami-iit/biomechanical-analysis-framework/pull/47/commits/28ee781a9cb364606ec986100072a1473c8acd84.